src/org/labkey/test/ClientErrorAssertionError.java

```diff
@@ -0,0 +1,9 @@
+package org.labkey.test;
+
+public class ClientErrorAssertionError extends AssertionError
+{
+    public ClientErrorAssertionError(Object detailMessage)
```

Comment:
AssertionError is very atypical of Throwable subclasses in taking an Object for its constructor. It really wants a String or a Throwable. I'd suggest using String as the argument type here.